README.md

```diff
@@ -115,4 +109,13 @@ The value of the kcidb_topic_name should be set to the env file
 echo "KCIDB_TOPIC_NAME=kcidb_new" >> .env
 ```
 
+## Services parameters
+
+### `trigger` service
 
+* --poll-period <seconds>: period to poll the git trees for new commits
+* --force: ignore if tree commit is already tested, keep testing on each poll-period
+* --build-configs: list of build configurations to test, separated by commas
+* --startup-delay <seconds>: delay before starting the first poll
+* --timeout <seconds>: timeout for the checkout node
+* --trees: list of trees to test, separated by commas. Prefix with '!' to exclude a tree, e.g. '!linux'. "!kernelci,!linux" will exclude kernelci and linux trees. "kernelci" will include only kernelci tree. Default is all trees.
```

Comment:
Shouldn't this section go to `pipeline-details` page?

Reply:
I removed this section totally, for now just removed dead links. I think we will redesign README files according best practices of python/github projects.